### PR TITLE
Fix associations/scope

### PR DESCRIPTION
**Pull request by dougmrqs**
Replaces #69.

### CODE REVIEW

File: tests/scope_test.js [hunk 1]
```diff
@@ -0,0 +1,577 @@
...
+        constraints: false
+      });
+      this.Comment.belongsTo(this.Question, {
+        foreignKey: 'commentable_id',
+        as: 'question',
+        constraints: false
+      });
+    });
+
+    // Reason: Fixed in another PR: https://github.com/cockroachdb/sequelize-cockroachdb/pull/64
```
Comment (RichardJCai) [inline tests/scope_test.js:96]: Are we able to unskip this if it was fixed?

File: tests/scope_test.js [hunk 2]
```diff
@@ -0,0 +1,577 @@
...
+
+        expect(post.coloredComments.length).to.equal(2);
+        for (const comment of post.coloredComments) {
+          expect(comment.type).to.match(/blue|green/);
+        }
+      });
+    });
+
+    describe('N:M', () => {
+      // Reason: Fixed in another PR: https://github.com/cockroachdb/sequelize-cockroachdb/pull/64
```
Comment (RichardJCai) [inline tests/scope_test.js:303]: Same with this

**Comment by RichardJCai** [comment]
https://github.com/cockroachdb/sequelize-cockroachdb/pull/72 was merged so I'm assuming we can unskip the two tests that depended on #64 which became #72

File: tests/scope_test.js (hunk 1 above)
Comment (dougmrqs) [reply tests/scope_test.js:96]: Probably. I'll check it out.

**Review by RichardJCai — APPROVED** [review]
Thanks!

**Pull request merged**